azure-pipelines.yml

```diff
@@ -12,4 +12,3 @@ pr:
 jobs:
   - template: ./.azure-pipelines/azure-pipelines-linux.yml
   - template: ./.azure-pipelines/azure-pipelines-osx.yml
-  - template: ./.azure-pipelines/azure-pipelines-win.yml
```

Comment:
this looks wrong.

Reply:
(Keep in mind this is my first recipe, so I'm bound to ask dumb questions)
 
What's the way to handle unsupported platforms then? Is it enough if no artefact is generated? Still wasteful of CI time though.

Reply:
yes, we waste some CI but only in the staged-recipes. This is taken care of in the generated "feedstock".

Merging this would destroy the pipeline for everyone coming after you.

Reply:
Oh I see, I'm sorry: I'll revert that when I get to the office!